hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/throttling/ThrottleDefValidationSuite.java

```diff
@@ -55,10 +55,11 @@ public List<HapiSpec> getSpecsInSuite() {
             throttleUpdateRejectsMultiGroupAssignment(),
             throttleUpdateWithZeroGroupOpsPerSecFails(),
             updateWithMissingTokenMintGetsWarning(),
-            ensureDefaultsRestored(),
+            ensureDefaultsRestored()
         });
     }
 
+    // @HapiTest failing due to charging fees not calculated properly
```

Comment:
Just out of curiosity, do you know why the fees are calculated incorrectly? I believe everything is in place by now.

Reply:
There were recent changes in the FileUpdateHandler file that added this code
```
final var fees = handleContext.feeCalculator(SubType.DEFAULT).legacyCalculate(sigValueObj -> {
    return new FileUpdateResourceUsage(fileOpsUsage)
            .usageGiven(fromPbj(handleContext.body()), sigValueObj, fromPbj(file));
});

handleContext.feeAccumulator().charge(handleContext.payer(), fees);
```
These changes were breaking the tests. After discussions in Slack/the modularization call(I forgot where exactly) we said that we would fix that.
Lev already has an open PR that fixes the issue: https://github.com/hashgraph/hedera-services/pull/8508/files#diff-69090fd45bc0fe76fc77b265659aeccf5339a25391220bfd927b6940246a7a50

So we should wait to merge this PR until Lev's PR is merged and I think 3-4 tests will pass. The last one is waiting on the authorization logic(I think you said that you will work on that).

Reply:
Now that Lev's PR is merged 4 out of 5 tests are failing with **INSUFFICIENT_PAYER_BALANCE**. This should be fixed when this is done - https://limechain.slack.com/archives/C040H9JPQT1/p1693206388283239. The 5th test is failing with **AUTHORIZATION_FAILED** which I think it's related to the same thing. I would say we can merge this PR that adds the missing validations and when the authorization is done we can revisit the tests and see if they are green or they need additional work.